OpenRA.Game/FieldSaver.cs

```diff
@@ -118,11 +107,12 @@ public static string FormatValue(object v)
 				return result;
 			}
 
+			// TODO
 			if (t.IsGenericType && t.GetGenericTypeDefinition() == typeof(Primitives.Cache<,>))
-				return ""; // TODO
+				return "";
```

Comment:
In case this is unused we should think about removing it instead of saving a placeholder value (by accident).

Reply:
Yes, lets remove it. I'm pretty sure that this isn't used anywhere near yaml serialization/loading, and if it is then we should fix that.